packages/office-ui-fabric-react/src/components/TextField/TextField.scss

```diff
@@ -106,13 +98,42 @@
       border-color: $ms-color-contrastWhiteSelected;
     }
   }
-
-  &[disabled] {
+  .rootIsDisabled > & {
     background-color: $ms-color-neutralLighter;
     border-color: $ms-color-neutralLighter;
     pointer-events: none;
     cursor: default;
   }
+}
+
+.fieldAddon {
+  background: #F4F4F4;
+  color: #666666;
```

Comment:
Hah...thanks for  catching that. Grabbed hex values from the designer, forgot to convert them.